diff -ur trunk.org/.waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gcc.py trunk/.waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gcc.py
--- trunk.org/waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gcc.py	Sat Apr 20 18:06:49 2019
+++ trunk/waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gcc.py	Sun Dec 18 14:14:07 2022
@@ -26,7 +26,7 @@
 	v.STLIBPATH_ST='-L%s'
 	v.RPATH_ST='-Wl,-rpath,%s'
 	v.SONAME_ST='-Wl,-h,%s'
-	v.SHLIB_MARKER='-Wl,-Bdynamic'
+	v.SHLIB_MARKER=''
 	v.STLIB_MARKER='-Wl,-Bstatic'
 	v.cprogram_PATTERN='%s'
 	v.CFLAGS_cshlib=['-fPIC']
diff -ur trunk.org/.waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gxx.py trunk/.waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gxx.py
--- trunk.org/waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gxx.py	Sat Apr 20 18:06:49 2019
+++ trunk/waf3-2.0.19-1f3c580272b15a03d2566843c5fe872a/waflib/Tools/gxx.py	Sun Dec 18 14:14:18 2022
@@ -26,7 +26,7 @@
 	v.STLIBPATH_ST='-L%s'
 	v.RPATH_ST='-Wl,-rpath,%s'
 	v.SONAME_ST='-Wl,-h,%s'
-	v.SHLIB_MARKER='-Wl,-Bdynamic'
+	v.SHLIB_MARKER=''
 	v.STLIB_MARKER='-Wl,-Bstatic'
 	v.cxxprogram_PATTERN='%s'
 	v.CXXFLAGS_cxxshlib=['-fPIC']

